saleor/static/js/components/categoryPage/ProductPrice.js

```diff
@@ -11,7 +13,7 @@ const ProductPrice = ({ availability, price }) => {
         {isPriceRange && <span>{gettext('from')} </span>} {gross}
       </span>
       {discount && (
-        <div className="product-list__sale"><span>Sale</span></div>
+        <div className="product-list__sale"><img src={SaleImg}/><span>Sale</span></div>
```

Comment:
We miss a translation here. It should be `{gettext('Sale')}`. See how we use `gettext` in JS in other modules for reference.